contracts/Nexus.sol

```diff
@@ -107,8 +107,10 @@ contract Nexus is INexus, BaseAccount, ExecutionHelper, ModuleManager, UUPSUpgra
         } else {
             PackedUserOperation memory userOp = op;
             userOp.signature = _enableMode(validator, op.signature);
+            // Ensure the module being enabled is a validator
+            if (!_isValidatorInstalled(validator)) return VALIDATION_FAILED;
```

Comment:
We should modify the wiki then https://github.com/bcnmy/nexus/wiki/Modules#module-enable-mode here it states that ANY MODULE can be installed using enableMode @Aboudjem

Reply:
right ! @filmakarov @livingrockrises can you confirm it is the intended behavior?

Reply:
Fil's PR states this in finality 
https://github.com/bcnmy/nexus/pull/112

Reply:
so should be closed?